2_Operations/how-we-work.md

```diff
@@ -4,7 +4,13 @@ Niteo is a distributed team spread around the World. Here's a quick overview of
 
 ## Communication
 
-Instant messaging is done through Slack on different channels (operations, support, development etc.). We have daily standups on Zoom, each morning, where everyone has a minute or two to say what they’re working on and if they need any help.
+Written communication can sometimes be misinterpreted and could lead to the other person's heartrate going up.
+A bad example of this can be: "Hey, do you have some time to talk?".
+If a senior Nitean sent such a message to a trialist, they might interpret it as "Oh no, I did something wrong" and start stressing out completely unnecessarily, since the senior Nitean just wanted to ask them if they have any dietary or other preferences for the upcoming IRL.
```

Comment:
Or imagine sending over `Hey, do you have some time to talk on Monday?`. There goes the weekend!
```suggestion
If a senior Nitean sent such a message to a trialist, they might interpret it as "Oh no, I did something wrong" and start stressing out completely unnecessarily, since the senior Nitean just wanted to ask them if they have any dietary or other preferences for the upcoming IRL. Or imagine sending over `Hey, do you have some time to talk on Monday?`. There goes the weekend!
```